accelerator/errors.h

```diff
@@ -63,6 +63,10 @@ typedef enum {
   /**< Fail to create TA object */
   SC_TA_NULL = 0x02 | SC_MODULE_TA | SC_SEVERITY_FATAL,
   /**< NULL TA objects */
+  SC_TA_FAILED_RESPONSE = 0x03 | SC_MODULE_TA | SC_SEVERITY_FATAL,
+  /**< Error in TA response */
+  SC_TA_TX_DESERIALIZE_FAILED = 0x04 | SC_MODULE_TA | SC_SEVERITY_FATAL,
```

Comment:
Are these status code still needed?

Reply:
We may need another issue to discuss about this

Reply:
Because these are not used in your modification XD

Reply:
Oh my